### PR TITLE
layout simple adaptative

### PR DESCRIPTION
**Pull request by abelhernando**

### CODE REVIEW

**Review by sergiomons — COMMENTED** [review]
Great job! although you can better it! keep going!

File: staff/AbelusM/css-layout/main.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+* {
+    padding: 0;
+    margin: 0;
+ }
+ 
+ html,
+ body {
+    min-height: 100% !important;
```
Comment (aperacaula) [inline staff/AbelusM/css-layout/main.css]: yew! no entiendo bien el porqué de mi-height y el !important. Que es lo que hace exactamente?

**Review by aperacaula — COMMENTED** [review]
Solo el detalle del !important que no entiendo, el resto de guais, lo hemos escrito casi igual!!

File: staff/AbelusM/css-layout/main.css (hunk 1 above)
Comment (abelhernando) [reply staff/AbelusM/css-layout/main.css]: is just for avoid to other stuff ever change the minimum height of the html or the body. the thing is !important makes sure of that instruction always will be the main option. 

es para tratar de evitar que cambie la altura mínima, el !important hace que eso siempre prevalezca para evitar modificarlo con otras instrucciones posteriores.

Comment (manuelbarzi) [reply staff/AbelusM/css-layout/main.css]: NOT required, trying removing it ,)

File: staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+
+* {
```
Comment (manuelbarzi) [inline staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css]: TIP move common style rules (mobile & desktop) to main.css

File: staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    background-color: black;
+    height: 100%;
+    width: 60%;
+    float: left;
+}
+footer {
+    border: 1px solid white;
+    background-color: black;
+    height: 20%;
+    width: 100%;
```
Comment (manuelbarzi) [inline staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css]: TIP is ```width: 100%```required?

File: staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+aside { 
+    border: 1px solid white;
+    background-color: black;
+    height: 100%;
+    width: 20%;
+    float: left;
+}   
+section {
+    border: 1px solid white;
+    background-color: black;
```
Comment (manuelbarzi) [inline staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css]: TIP group selectors that share same rules (i.e., group shared rules in group of selectors)

SEE Grouping Selectors https://www.w3schools.com/css/css_syntax.asp

File: staff/AbelusM/css-layout/main.css (hunk 1 above)
Comment (manuelbarzi) [reply staff/AbelusM/css-layout/main.css]: NOT required

**Comment by abelhernando** [comment]
I did both homework exercices without internet, that's why i had no proper images for App-Template, i did the structure but not the fancy stuff :(

**Comment by manuelbarzi** [comment]
ATTEND comments and apply changes please! ,)

File: staff/AbelusM/css-layout/main.css (hunk 1 above)
Comment (abelhernando) [reply staff/AbelusM/css-layout/main.css]: Agree , totally not necessary

File: staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css (hunk 3 above)
Comment (abelhernando) [reply staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css]: Not required, changed.

File: staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css (hunk 2 above)
Comment (abelhernando) [reply staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css]: done!

File: staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css (hunk 4 above)
Comment (abelhernando) [reply staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css]: thank you, done.

File: staff/AbelusM/js/challenges/count-words/scripts/countWords.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+function countWords(str) {
```
Comment (agandia9) [inline staff/AbelusM/js/challenges/count-words/scripts/countWords.js]: use strict?

**Review by agandia9 — COMMENTED** [review]
ok!

**Pull request merged**